packages/webdriverio/src/utils.js

```diff
@@ -370,3 +370,11 @@ export async function findElements(selector) {
 
     throw INVALID_SELECTOR_ERROR
 }
+
+/**
+ * Strip element object of all propeties except `elementId`
+ */
+
+export function verifyArgsAndStripIfElement(args) {
+    return args.map(a => a.elementId ? { elementId: a.elementId } : a)
```

Comment:
This doesn't work. No browser consumes the elementId property. It needs to be ELEMENT, or the jsonwp version.

Reply:
import { ELEMENT_KEY } from '../../constants'

{
  [ELEMENT_KEY]: this.elementId, // w3c compatible
  ELEMENT: this.elementId // jsonwp compatible
}

Reply:
Yea, this makes sense. I'm still very new to all this w3c and jsonwp compatibility, I appreciate your feedback. I will try to adjust the code based on your comment and the suggestion in the review. Cheers.

Reply:
@abjerstedt Before I push anything, I want to be sure that I understood you correctly.

These are the tests I added based on your comments, is this what you meant?

```javascript
describe('verifyArgsAndStripIfElement', () => {
    it('returns the same value if it is not an element object', () => {
        expect(verifyArgsAndStripIfElement([1, 2, 3])).toEqual([1, 2, 3])
    })

    it('strips down properties if value is element object', () => {
        const fakeObj = new (class Element {
            constructor() {
                this.elementId = 'foo-bar'
                this.someProp = 123
                this.anotherProp = 'abc'
            }
        })()

        expect(verifyArgsAndStripIfElement([fakeObj, 'abc', 123])).toMatchObject([
            { [ELEMENT_KEY]: 'foo-bar', ELEMENT: 'foo-bar' },
            'abc',
            123
        ])
    })

    it('should work even if parameter is not of type Array', () => {
        const fakeObj = new (class Element {
            constructor() {
                this.elementId = 'foo-bar'
                this.someProp = 123
                this.anotherProps = 'abc'
            }
        })()

        expect(verifyArgsAndStripIfElement(fakeObj)).toMatchObject(
            { [ELEMENT_KEY]: 'foo-bar', ELEMENT: 'foo-bar' }
        )
        expect(verifyArgsAndStripIfElement('foo')).toEqual('foo')
    })

    it('throws error if element object is missing element id', () => {
        const fakeObj = new (class Element {
            constructor() {
                this.someProp = 123
                this.anotherProps = 'abc'
            }
        })()

        expect(() => verifyArgsAndStripIfElement(fakeObj)).toThrow('property "elementId" missing')
    })
})
```

I'm sure using `new (class Element {})()` is not ideal, but I don't know how else to mimic the class object in order to have `if (arg.constructor.name === 'Element')` in the function

Reply:
I think this looks fine, I would define the class once at the top of the file though instead of redefining it in every test.